src/hooks/screen/useChatScreenHook.ts

```diff
@@ -23,6 +23,7 @@ import {
 } from '../../utils/constants';
 import {getAnonymousUserId, getUserId} from '../../utils/users';
 import {randomString} from '../../utils/string/StringUtils';
+import ImageUtils from '../../utils/image';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `ImageUtils` has been added. This is necessary for the new functionality of handling attachments in chat messages.
```diff
+ import ImageUtils from '../../utils/image';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->